bg96_gps.device.lib.nut

```diff
@@ -391,11 +391,11 @@ BG96_GPS <- {
         if (_session != null) {
             local t = _session.assist.read();
             local valid = (t.status == 0);
-            if ("xtradatadurtime" in t && t.xtradatadurtime == 0) valid = false;
-            local msg = valid ? "GNSS assist data valid" : "GNSS assist data invalid";
             local data = {};
+            data.time <- _getValidTime(t.injecteddatatime,t.xtradatadurtime);
+            valid = data.time > 0
```

Comment:
missing trailing semicolon is not mandatory for squirrel but would match the style of the existing code